locale/loris.pot

```diff
@@ -145,15 +146,23 @@ msgstr ""
 msgid "Site"
 msgstr ""
 
+msgid "Sites"
+msgstr ""
+
 msgid "Module"
 msgstr ""
 
 msgid "Project"
 msgstr ""
 
+msgid "Projects"
+msgstr ""
+
 msgid "Cohort"
-msgid_plural "Cohorts"
-msgstr[0] ""
+msgstr ""
+
+msgid "Cohorts"
+msgstr ""
```

Comment:
After https://github.com/aces/Loris/pull/10024 this is the new from of deal with plurals. Please take a look in 10024 and do the same for every plural